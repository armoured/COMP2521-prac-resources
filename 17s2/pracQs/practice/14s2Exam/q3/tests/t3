./q3 < tests/t3.in
