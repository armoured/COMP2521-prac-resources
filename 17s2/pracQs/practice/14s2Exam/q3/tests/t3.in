6
0-1
0-2
0-3
2-3
2-4
2-5
