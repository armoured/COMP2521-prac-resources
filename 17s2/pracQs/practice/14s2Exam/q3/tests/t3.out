Graph:
# vertices = 6, # edges = 6

0 is connected to 1 2 3
1 is connected to 0
2 is connected to 0 3 4 5
3 is connected to 0 2
4 is connected to 2
5 is connected to 2

Well-connected vertices
2 has 4 connections
0 has 3 connections
3 has 2 connections
